image/sigmiod_correction/test.py

```diff
@@ -0,0 +1,31 @@
+import numpy as np
```

Comment:
Always good to add a header block. Block would include authorship, license, copyright, dates and description of what the driver does.